### PR TITLE
[CARBONDATA-3791] Correct spelling, query, default value, in performance-tuning, prestodb and prestosql documentation.

### PR DESCRIPTION
**Pull request by nihal0107**
### Why is this PR needed?
 Correct spelling, query, default value, in performance-tuning, prestodb and prestosql documentation.
 
 ### What changes were proposed in this PR?
 Corrected spelling, query, default value, in performance-tuning, prestodb and prestosql documentation.
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
@nihal0107 : I have pushed the presto changes my self in their PR, rebase and handle other comments

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1201/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2919/

File: docs/performance-tuning.md [hunk 1]
```diff
@@ -128,6 +128,9 @@
 
   **NOTE:**
   + BloomFilter can be created to enhance performance for queries with precise equal/in conditions. You can find more information about it in BloomFilter index [document](./index/bloomfilter-index-guide.md).
+  + Lucine index can be created on string columns which has content of more length to enhance the query performance. You can find more information about it in Lucene index [document](./index/lucene-index-guide.md).
```
Comment (chetandb) [inline docs/performance-tuning.md:131]: Change "Lucine" spelling to "Lucene"

File: docs/performance-tuning.md [hunk 2]
```diff
@@ -141,12 +144,12 @@
...
-|carbon.sort.file.write.buffer.size|Default:  16384.|CarbonData sorts and writes data to intermediate files to limit the memory usage. This configuration determines the buffer size to be used for reading and writing such files. |
+|carbon.sort.size|Default: 100000. The value should be >= 1000.|Threshold to write local file in sort step when loading data|
+|carbon.sort.file.write.buffer.size|Default:  16384. The value should be >= 10240 and <= 10485760.|CarbonData sorts and writes data to intermediate files to limit the memory usage. This configuration determines the buffer size to be used for reading and writing such files. |
 |carbon.merge.sort.reader.thread|Default: 3 |Specifies the number of cores used for temp file merging during data loading in CarbonData.|
 |carbon.merge.sort.prefetch|Default: true | You may want set this value to false if you have not enough memory|
 
-  For example, if there are 10 million records, and i have only 16 cores, 64GB memory, will be loaded to CarbonData table.
+  For example, if there are 10 million records, and I have only 16 cores, 64 GB memory, will be loaded to CarbonData table.
```
Comment (chetandb) [inline docs/performance-tuning.md:152]: Remove comma in 64 GB memory,. Change to "For example, if there are 10 million records, and I have only 16 cores, 64 GB memory will be loaded to CarbonData table."

File: docs/prestodb-guide.md [hunk 3]
```diff
@@ -139,11 +139,14 @@ Then, `query.max-memory=<30GB * number of nodes>`.
 
 ##### Configuring Carbondata in Presto
 1. Create a file named `carbondata.properties` in the `catalog` folder and set the required properties on all the nodes.
+2. As carbondata connector extends, hive connector. All the configurations(including S3) is same as hive connector.
```
Comment (chetandb) [inline docs/prestodb-guide.md:142]: " As carbondata connector extends, hive connector. All the configurations(including S3) is same as hive connector." can be changed to " As carbondata connector extends hive connector all the configurations(including S3) is same as hive connector."

File: docs/prestodb-guide.md [hunk 4]
```diff
@@ -139,11 +139,14 @@ Then, `query.max-memory=<30GB * number of nodes>`.
...
+Just replace the connector name in hive configuration and make a copy of it to carbondata.properties
```
Comment (chetandb) [inline docs/prestodb-guide.md:143]: "make a copy of it to carbondata.properties" can be changed to "copy same to carbondata.properties"

File: docs/prestosql-guide.md [hunk 5]
```diff
@@ -139,11 +139,15 @@ Then, `query.max-memory=<30GB * number of nodes>`.
 
 ##### Configuring Carbondata in Presto
 1. Create a file named `carbondata.properties` in the `catalog` folder and set the required properties on all the nodes.
+2. As carbondata connector extends, hive connector. All the configurations(including S3) is same as hive connector.
```
Comment (chetandb) [inline docs/prestosql-guide.md:142]: "As carbondata connector extends, hive connector. All the configurations(including S3) is same as hive connector."  can be changed to "As carbondata connector extends hive connector all the configurations(including S3) is same as hive connector."

File: docs/prestosql-guide.md [hunk 6]
```diff
@@ -139,11 +139,15 @@ Then, `query.max-memory=<30GB * number of nodes>`.
...
+Just replace the connector name in hive configuration and make a copy of it to carbondata.properties
+`connector.name = carbondata`
```
Comment (chetandb) [inline docs/prestosql-guide.md:144]: "make a copy of it to carbondata.properties" can be changed to "copy same to carbondata.properties"

Comment (nihal0107) [reply docs/prestosql-guide.md:144]: done

File: docs/prestosql-guide.md (hunk 5 above)
Comment (nihal0107) [reply docs/prestosql-guide.md]: done

File: docs/performance-tuning.md (hunk 1 above)
Comment (nihal0107) [reply docs/performance-tuning.md]: done

File: docs/performance-tuning.md (hunk 2 above)
Comment (nihal0107) [reply docs/performance-tuning.md]: done

File: docs/prestodb-guide.md (hunk 3 above)
Comment (nihal0107) [reply docs/prestodb-guide.md]: done

File: docs/prestodb-guide.md (hunk 4 above)
Comment (nihal0107) [reply docs/prestodb-guide.md]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2932/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1214/

**Comment by ajantha-bhat** [comment]
LGTM

**Pull request closed without merging**